wfl/cli/cli.py

```diff
@@ -1,56 +1,15 @@
-"""
-Command line interface to the package
-"""
-
-import distutils.util
-import glob
-import json
-import os
-import sys
 import warnings
-from pprint import pformat, pprint
-from pathlib import Path
 
-import ase.data
-import ase.io
 import click
-import numpy as np
-import yaml
-
-try:
-    import quippy
-except ModuleNotFoundError:
-    pass
-# noinspection PyProtectedMember
-from ase.io.extxyz import key_val_str_to_dict
-
-from wfl.configset import ConfigSet, OutputSpec
-from wfl.generate import normal_modes as nm 
-import wfl.generate.smiles
-import wfl.utils.misc
-import wfl.generate.buildcell
-import wfl.select.by_descriptor
-import wfl.descriptors.quippy
-
-from wfl.utils import gap_xml_tools
+from wfl.cli import cli_options as opt
```

Comment:
Is this actually used in the top level `cli.py`, or only the actual subgroups?

Reply:
just subgroups, removed it from here.